packages/web/src/components/nav/desktop/PlaylistLibrary.tsx

```diff
@@ -42,6 +44,47 @@ type PlaylistLibraryProps = {
   onClickNavLinkWithAccount: () => void
 }
 
+type LibraryContentsLevelProps = {
+  contents: PlaylistLibraryType['contents']
+  renderPlaylist: (playlistId: number) => void
+  renderExplorePlaylist: (playlistId: SmartCollectionVariant) => void
+  renderFolder: (folder: PlaylistLibraryFolder) => void
+}
+
+// Function component for rendering a single level of the playlist library. Playlist library consists of up to two content levels (root + inside a folder)
```

Comment:
super ultra nit, but let's do component comments in format

```
/*
 *
 */
```

https://www.notion.so/audiusproject/Client-Javascript-Styleguide-4f34640139fc4a809c3765131bc597ff#5ec92a9e0f9741a8a61e5b71d528b74c